Justfile

```diff
@@ -46,8 +46,20 @@ install-dev:
 clean:
     rm -rf .pytest_cache .ruff_cache build dist *.egg-info **/__pycache__
 
-# -------------------- Project-specific tasks --------------------
+# -------------------- Ameren Data Collection --------------------
 
-# Download all Ameren CSVs (your Selenium+requests script)
+# Download Ameren CSV files and upload to S3 (interactive mode)
+# Note: May require 2-3 runs due to server rate limiting
+# The script automatically skips files already in S3
 download-ameren:
```

Comment:
What does the scraper do if you don't supply a bucket? Download to local? Clarify in the comment.